### DISCUSSION TITLE
does not connect any agent

### DISCUSSION THREAD

**Discussion by DFBG7** [wazuh/wazuh#18755] [Q&A]
![wazuh](https://github.com/wazuh/wazuh/assets/143821506/4f7dbaf7-dfd6-4720-b5bc-187b7171ba44)

Can someone guide me on this problem?

**Comment by stevenbowyer** [comment]
Based on this link:

https://documentation.wazuh.com/current/user-manual/reference/tools/agent-auth.html

Shouldn't the command be:

`.\agent-auth.exe -a -m 10.107.2.20`

Assuming the manager IP is 10.107.2.20?
